packages/components/link/README.md

```diff
@@ -47,7 +47,7 @@ export default Example;
 | Props         | Type                                                           | Required | Default     | Description                                                                                                                                                          |
 | ------------- | -------------------------------------------------------------- | :------: | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `children`    | `ReactNode`                                                    |          |             | Value of the link.&#xA;<br />&#xA;Required if `intlMessage` is not provided.                                                                                         |
-| `intlMessage` | `MessageDescriptor`                                            |          |             | An `intl` message object that will be rendered with `FormattedMessage`.&#xA;<br />&#xA;Required if `children` is not provided.                                       |
+| `intlMessage` | `intersection`                                                 |          |             | An `intl` message object that will be rendered with `FormattedMessage`.&#xA;<br />&#xA;Required if `children` is not provided.                                       |
```

Comment:
After extending the `MessageDescriptor` type in the definition, now we have this not-so-meaningful value in the READMEs.
Could use some help to figure out the better way to address this.